.github/workflows/build.yml

```diff
@@ -108,11 +164,13 @@ jobs:
         run: NDK_CCACHE=1 NDK_CCACHE_BIN=ccache ./gradlew app:assemble${{ matrix.build_type }} --info
 
   build_ios:
+    # iOS is 10x expensive to run on GitHub machines, so only run if we know something else fast/simple passed as well
+    needs: build
     name: "Build ios in ${{ matrix.build_type }}"
     runs-on: macos-latest
     strategy:
       matrix:
-        build_type: [Debug]
+        build_type: [Release]
```

Comment:
Release builds are about 3 times faster, this build is currently around 20+ minutes and should reduce it greatly